lib/features/sks-menu/presentation/widgets/sks_menu_loading.dart

```diff
@@ -0,0 +1,70 @@
+import "package:flutter/material.dart";
+import "package:lottie/lottie.dart";
+
+import "../../../../../theme/app_theme.dart";
+import "../../../../../utils/context_extensions.dart";
+import "../../../../gen/assets.gen.dart";
+
+class LottieLoading extends StatefulWidget {
+  const LottieLoading({
+    super.key,
+  });
+
+  @override
+  _LottieLoadingState createState() => _LottieLoadingState();
+}
+
+class _LottieLoadingState extends State<LottieLoading> {
+  final bool _showOldMenu = false;
+
+  void _gotoOldMenu() {
+    //Navigator.push(
+    //context,
+    //MaterialPageRoute(builder: (context) => const _SksMenuView()),
+    // );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          const Spacer(),
+          Center(
+            child: SizedBox(
+              width: 200,
+              height: 200,
+              child: Lottie.asset(
+                Assets.animations.closed,
+                repeat: false,
+              ),
+            ),
+          ),
+          const Spacer(),
+          Align(
```

Comment:
As this tasks's scope was reduced, we don;t need such a button here. 

But it would be great if some message's displayed when error occured. Such us, menu is not available, pls try again later, or sth like that. So that user could understand what's going on